hoss/hoss_config.json

```diff
@@ -182,27 +164,135 @@
           },
           "Attributes": [
             {
-              "Name": "Grid_Mapping",
-              "Value": "EASE2_Polar"
+              "Name": "grid_mapping",
+              "Value": "/EASE2_polar_projection"
             }
           ],
           "_Description": "Some versions of these collections omit polar grid mapping information"
         }
       ]
     },
+    {
+        "Applicability": {
+          "Mission": "SMAP",
+          "ShortNamePath": "SPL3SMP_E"
+        },
+        "Applicability_Group": [
+          {
+            "Applicability": {
+              "Variable_Pattern": "Soil_Moisture_Retrieval_Data_(A|P)M/.*"
+            },
+            "Attributes": [
+              {
+                "Name": "grid_mapping",
+                "Value": "/EASE2_global_projection"
+              }
+            ],
+            "_Description": "Some versions of these collections omit global grid mapping information"
+          },
+          {
+            "Applicability": {
+              "Variable_Pattern": "Soil_Moisture_Retrieval_Data_Polar_(A|P)M/.*"
+            },
+            "Attributes": [
+              {
+                "Name": "grid_mapping",
+                "Value": "/EASE2_polar_projection"
+              }
+            ],
+            "_Description": "Some versions of these collections omit polar grid mapping information"
+          }
+        ]
+    },
     {
       "Applicability": {
         "Mission": "SMAP",
         "ShortNamePath": "SPL3FTA"
       },
       "Attributes": [
         {
-          "Name": "Grid_Mapping",
-          "Value": "EASE2_Polar"
+          "Name": "grid_mapping",
+          "Value": "/EASE2_polar_projection"
         }
       ],
       "_Description": "Some versions of these collections omit polar grid mapping information"
     },
+    {
+        "Applicability": {
+          "Mission": "SMAP",
+          "ShortNamePath": "SPL3SM(P|A|AP)|SPL2SMAP_S"
+        },
+        "Attributes": [
+          {
+            "Name": "grid_mapping",
+            "Value": "/EASE2_global_projection"
+          }
+        ],
+        "_Description": "Some versions of these collections omit global grid mapping information"
+    },
+    {
+        "Applicability": {
+          "Mission": "SMAP",
+          "ShortNamePath": "SPL3FT(P|P_E)|SPL3SM(P|P_E|A|AP)|SPL2SMAP_S"
+        },
+        "Applicability_Group": [
```

Comment:
I don't see this comment addressed in the latest commit of the PR.

Reply:
Looks like this has now been addressed - thank you!

One thing I do see now, though, is a mixture of indentation. Prior to this commit, the JSON file adopted indentation of 2 spaces per level. There is now a mixture of 2 and 4 spaces. It would be good to keep the file consistent, to assist with people reading the file.